scipy/stats/tests/test_stats.py

```diff
@@ -6397,7 +6397,8 @@ def test_against_R(self, alternative, xp):
         # test_result@test$p.value
         test_name = self.test_name
         test_fun = getattr(stats, test_name)
-        ref_statistic, ref_pvalue = xp.asarray(self.case_ref)
+        xp_case_ref = xp.asarray(self.case_ref)
+        ref_statistic, ref_pvalue = xp_case_ref[0], xp_case_ref[1]
```

Comment:
Just for consistency with what was done above. Could go either way... this way is probably less efficient, but doesn't need an intermediate variable. 
```suggestion
        ref_statistic= xp.asarray(self.case_ref[0])
        ref_pvalue = xp.asarray(self.case_ref[1])
```
I suppose this create NumPy 0d arrays whereas the one above is NumPy scalars, but doesn't seem to matter.